app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
Ok, how about @h-kataria @himdel... My comment is that it feels like we should be doing this "convert the Vm class to VM since it's an abbreviation" in the UI in many other places.  I'm fine with doing it here but would like to have an issue opened to refactor this pattern into a display method in case this is done in many places.

Once this is merged, we won't look at it.